packages/core/src/indexing/postgres/init-tables.ts

```diff
@@ -34,10 +45,24 @@ function relationsDefinitionsToColumnInfo(relations?: ModelRelationsDefinition):
   })
 }
 
+/**
+ * Create Compose DB config tables
+ */
+export async function initConfigTables(dataSource: Knex, logger: DiagnosticsLogger) {
+  const configTables = await listConfigTables()
+  for (const table of configTables) {
+    const exists = await dataSource.schema.hasTable(table.tableName)
+    if (!exists) {
+      logger.imp(`Creating Compose DB config table: ${table.tableName}`)
+      await createConfigTable(dataSource, table.tableName)
```

Comment:
Wanted to create the table with the `CONFIG_TABLE_MODEL_INDEX_STRUCTURE` object and looked into `knex` but unfortunately no support for that - would've been nice to use the `.columInfo()` dump to do so. There is a method called `knex.schema.createTableLike` but that only takes another table as input.